modules/dynamic-provisioning-storage-class-definition.adoc

```diff
@@ -33,4 +33,4 @@ parameters: <6>
 <4> (optional) Annotations for the storage class.
 <5> (required) The type of provisioner associated with this storage class.
 <6> (optional) The parameters required for the specific provisioner, this
```

Comment:
Want to undo that hard wrap while you're in here?
```suggestion
<6> (optional) The parameters required for the specific provisioner, this will change from plug-in to plug-in.
```

Reply:
good idea!